.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "useTabs": false,
+  "printWidth": 120,
+  "tabWidth": 2,
+  "singleQuote": true,
+  "semi": false
```

Comment:
Keep `semi: true`.
Also, you have removed some properties from `.prettierrc ` which was present before in your PR.